dfget/core/downloader/back_downloader/back_downloader.go

```diff
@@ -129,3 +133,7 @@ func (bd *BackDownloader) Cleanup() {
 	}
 	bd.cleaned = true
 }
+
+func (bd *BackDownloader) isSuccessStatus(code int) bool {
+	return code < 400
```

Comment:
`code < 400` -> `code <= 400`?

Reply:
`400` is also an error code caused by user

Reply:
Sorry for my mistake that I consider the opposite. 😢

Reply:
This is not enough. Only 200~299  is considered as success. HTTP espone codes:
1xx : continue
2xx: success
3xx: redirect
4xx: client errors
5xx: server internal errors